### PR TITLE
Make GCloud artifacts page show up in Prow directly

### PR DESCRIPTION
**Pull request by msugakov**
## Description

Also style this and other pages like Prow.

## Checklist
- [x] Investigated and inspected CI test results

None of these are needed:
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

- [x] Checked how the output looks in Prow and shared the screenshot in this PR.

### CODE REVIEW

**Comment by msugakov** [comment]
An early look (before adjustment) is available here https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/stackrox_stackrox/5603/pull-ci-stackrox-stackrox-master-ocp-4-10-operator-e2e-tests/1644055638038286336

Waiting for more recent results.

File: scripts/ci/store-artifacts.sh [hunk 1]
```diff
@@ -136,29 +136,47 @@ make_artifacts_help() {
...
+        <title><h4>How to locate other artifacts</h4></title>
+        <style>
+          body { color: #e8e8e8; background-color: #424242; font-family: "Roboto", "Helvetica", "Arial", sans-serif }
+          a { color: #ff8caa }
+          a:visited { color: #ff8caa }
+        </style>
+        </head>
+        <body>
+
         Artifacts are stored in a GCS bucket ($GS_URL). There are at least two options for access:
```
Comment (gavin-stackrox) [inline scripts/ci/store-artifacts.sh:156]:
```suggestion
        Additional stackrox e2e artifacts are stored in a GCS bucket ($GS_URL) by the store_artifacts bash function. There are at least two options for access:
```

**Review by gavin-stackrox — APPROVED** [review]
LGTM with some additional suggestions.

**Comment by msugakov** [comment]
image

**Pull request merged**